src/modules/Sticky/Sticky.js

```diff
@@ -66,11 +66,15 @@ export default class Sticky extends Component {
 
     /** Whether element should be "pushed" by the viewport, attaching to the bottom of the screen when scrolling up. */
     pushing: PropTypes.bool,
+
+    /** Context which sticky should attach `onscroll` events. */
```

Comment:
I'm using the same descriptions from the SemanticUI docs. I'll use this one, without backticks, for both